### DISCUSSION TITLE
sudo mount -t virtiofs share /Volumes/stuff

### DISCUSSION THREAD

**Discussion by vamsi3379** [utmapp/UTM#6152] [Linux]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
I'm not able to use shared folder (which is on Mac) in UTM Linux OS.

**Configuration**  
* UTM Version: Latest( downloded 2 days ago) 
* macOS Version: Ventura 13.5.2 
* Apple Silicon Chip M2 Pro

**Crash log**  
<img width="915" alt="Screenshot 2023-10-03 at 3 50 58 PM" src="https://github.com/utmapp/UTM/assets/48962308/dca85553-5cd5-4679-b1d4-f4ae4e7b2519">

<img width="901" alt="Screenshot 2023-10-03 at 3 54 59 PM" src="https://github.com/utmapp/UTM/assets/48962308/efdaeead-fb0d-44f8-b83e-521672d0c92a">

**Debug log**  

**Upload VM**  
<img width="937" alt="Screenshot 2023-10-03 at 3 51 10 PM" src="https://github.com/utmapp/UTM/assets/48962308/9a14220d-3655-4efb-be01-b60abf09f919">

**Comment by jufiba** [comment]
Have the same problem with the latest Sonoma 14.0, UTM 4.3.5 and Ubuntu 23.04 (arm64) as guest. Which used to work before I upgraded everything. Regrettably I did not check after upgrading each item.

**Comment by jufiba** [comment]
Using the newest beta UTM version (4.4.3) it works nicely.

Best,
Juan

**Comment by light-idea** [comment]
I also had the same issue on Sonoma 14.0, UTM 4.3.5 with a Fedora 38 guest.  Virtiofs works again after updating UTM to 4.4.4.
